docs/framework/additional-apis/_httpresponse.md

```diff
@@ -0,0 +1,33 @@
+---
+title: "_HttpResponse Field | Microsoft Docs"
+ms.date: "04/24/2017"
+ms.prod: ".net-framework"
+ms.technology: ""
+ms.topic: "reference"
+topic_type: "apiref"
+api_name:
+  - "System.Net.HttpWebRequest._HttpResponse"
+api_location:
+  - "System.dll"
+api_type: "Assembly"
+ms.assetid: eab9b789-beb4-4c28-b2d8-78debc7ba129
+author: "guardrex"
+ms.author: "mairaw"
+manager: "wpickett"
+---
+# \_HttpResponse Field
```

Comment:
For the member pages, I'm thinking we should add the class name to the title too similar to an automated generated page: https://docs.microsoft.com/en-us/dotnet/api/microsoft.visualbasic.constants.vbapplicationmodal?view=netframework-4.6.2